### PR TITLE
Excercising Gatsby Cloud previews!

### PR DESCRIPTION
**Pull request by merit-mthompson**
Not making any content changes, just a README note about the preview generation. We should all see a comment after this that links to a preview copy of the site.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ You can also browse GraphQL at `http://localhost:8000/__graphql`.
 
 Deploys are handled automatically by GitHub Actions whenever a branch or PR is merged and visible at <https://merit-network.github.io/>. After a deploy runs, please allow a minute or two for GitHub Pages to catch up.
 
+When you open a PR a preview will be automatically created and shared in as a PR comment.
```
Comment (merit-mthompson) [inline README.md:36]:
```suggestion
When you open a PR a preview will be automatically created and shared as a PR comment.
```

**Comment by merit-mthompson** [comment]
Previews are not regenerated when we change content via PR suggestions or when we make changes directly and update the branch. For now, as a PR evolves, we will have to manually re-run preview generation whenever content changes.

**Comment by merit-mthompson** [comment]
Sorry, looks like builds are re-run on every commit. The comment above is also edited by the Gatsby Cloud bot whenever we make changes.

**Review by merit-schelcj — APPROVED** [review]

**Review by merit-dsclair — APPROVED** [review]
Good to go! Looks great!

**Pull request merged**